### PR TITLE
DOMA-2987 fix admin ui on build phaze

### PR DESCRIPTION
**Pull request by dkoviazin**
Register all schema in build phase (if not - they will not appear in admin ui)

### CODE REVIEW

File: apps/condo/admin-ui/index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// @ts-ignore
```
Comment (pahaz) [inline apps/condo/admin-ui/index.js:1]: do we really need it for js files?

File: apps/condo/index.js [hunk 2]
```diff
@@ -67,33 +67,30 @@ const keystone = new Keystone({
...
 // They can be safely ignored without impact on final executable code
-if (IS_BUILD_PHASE) {
-    // `User` list is used by `createAuthStrategy`, that creates `AuthStrategy`, required to build /admin/ static files
-    registerSchemas(keystone, [
-        require('@core/keystone/schemas/User'),
-    ])
-} else {
+
+// We need to register all schemas as they will appear in admin ui
+registerSchemas(keystone, [
```
Comment (pahaz) [inline apps/condo/index.js:72]: do we really need all these domains on the build stage?

**Review by pahaz — APPROVED** [review]

File: apps/condo/admin-ui/index.js (hunk 1 above)
Comment (dkoviazin) [reply apps/condo/admin-ui/index.js:1]: git hub warnings :-)
image

**Review by sitozzz — APPROVED** [review]

File: apps/condo/admin-ui/index.js (hunk 1 above)
Comment (pahaz) [reply apps/condo/admin-ui/index.js:1]: :)) GitHub think that we use TS

File: apps/condo/index.js (hunk 2 above)
Comment (pahaz) [reply apps/condo/index.js:72]: Omg, thank.

**Review by pahaz — APPROVED** [review]

**Pull request merged**